src/bp/ui-studio/src/web/views/OneFlow/sidePanel/GoalEditor/index.tsx

```diff
@@ -107,92 +83,58 @@ const EditGoalModal: FC<Props> = props => {
+              onChange={e => setName(sanitizeName(e.currentTarget.value))}
+              autoFocus
             />
-
-            <Tab
-              id="overview"
-              title="Overview"
-              className={style.tabs}
-              panel={
-                <div>
-                  <FormGroup label="Name" helperText="The name is used internally">
-                    <InputGroup
-                      id="input-flow-name"
-                      tabIndex={1}
-                      required
-                      value={name || ''}
-                      onChange={e => setName(sanitizeName(e.currentTarget.value))}
-                      autoFocus
-                    />
-                  </FormGroup>
-
-                  <FormGroup
-                    label="Label"
-                    helperText="The label is a friendly name that can replace the name in the topic list"
-                  >
-                    <InputGroup
-                      id="input-flow-label"
-                      tabIndex={2}
-                      value={label || ''}
-                      onChange={e => setLabel(e.currentTarget.value)}
-                    />
-                  </FormGroup>
-
-                  <FormGroup label="Description">
-                    <TextArea
-                      id="input-flow-description"
-                      rows={3}
-                      tabIndex={3}
-                      value={description || ''}
-                      fill
-                      onChange={e => setDescription(e.currentTarget.value)}
-                    />
-                  </FormGroup>
-
-                  <Button
-                    type="submit"
-                    id="btn-submit"
-                    tabIndex={4}
-                    text="Save changes"
-                    intent={Intent.PRIMARY}
-                    className={style.modalFooter}
-                  />
-                </div>
-              }
+          </FormGroup>
+
+          <FormGroup
+            label="Label"
+            helperText="The label is a friendly name that can replace the name in the topic list"
+          >
+            <InputGroup
+              id="input-flow-label"
+              tabIndex={2}
+              value={label || ''}
+              onChange={e => setLabel(e.currentTarget.value)}
+            />
+          </FormGroup>
+
+          <FormGroup label="Description">
+            <TextArea
+              id="input-flow-description"
+              rows={3}
+              tabIndex={3}
+              value={description || ''}
+              fill
+              onChange={e => setDescription(e.currentTarget.value)}
             />
-          </Tabs>
+          </FormGroup>
+
+          <Button
+            type="submit"
+            id="btn-submit"
+            tabIndex={4}
+            text="Save changes"
+            intent={Intent.PRIMARY}
+            className={style.modalFooter}
+          />
         </div>
       </DialogBody>
```

Comment:
There is a DialogFooter component that you can put after DialogBody for the buttons